CHANGELOG.md

```diff
@@ -1,5 +1,21 @@
 # Changelog
 
+## [2.16.1](https://github.com/puppetlabs/beaker-pe/tree/2.16.1) (2024-03-19)
```

Comment:
I think we want 3.0.1 here along with today's date.

Reply:
Thanks @steveax, I have corrected that.
Although, the CHANGELOG was auto-generated by the release-prep script, I also noticed that the github action to release the gem prevents the release in case the logs are not the same as the release-prep script would generate.
Like in this run: https://github.com/puppetlabs/beaker-pe/actions/runs/8551160057/job/23429573751

Not sure if I am missing something. Any thoughts?

Reply:
Ughh, so I guess the changelog generator doesn't know how to deal with multiple branches? I'm about this '' close to ripping all that out.

Reply:
OK, I looked at the action and I _think_ perhaps the check will give the thumbs up.

Reply:
Oh I see. I'll give it a try after this is merged and see if it proceeds. Thanks!